internals/plan/plan.go

```diff
@@ -57,6 +104,19 @@ type Layer struct {
 	Services    map[string]*Service   `yaml:"services,omitempty"`
 	Checks      map[string]*Check     `yaml:"checks,omitempty"`
 	LogTargets  map[string]*LogTarget `yaml:"log-targets,omitempty"`
+
+	Sections map[string]LayerSection `yaml:",inline,omitempty"`
+}
+
+// addSection adds a new section to the layer.
+func (layer *Layer) addSection(field string, section LayerSection) {
+	layer.Sections[field] = section
+}
+
+// Section retrieves a layer section from a layer. Returns nil if
+// the field does not exist.
+func (layer *Layer) Section(field string) LayerSection {
+	return layer.Sections[field]
 }
```

Comment:
`Layer.Sections` being unexported would justify having a `Section(string)` getter. However, this method is equivalent to just indexing `layer.Sections`. Also, `layer.Sections[field]` reads obvious and clear, why bother adding a getter?

Reply:
I've removed this, essentially inlining it in placed its used.

Reply:
Resolving.